server/app/views/v1/grid_service_instances/_grid_service_instance.json.jbuilder

```diff
@@ -0,0 +1,16 @@
+json.id service_instance.id.to_s
```

Comment:
This uses the mongo ID as the API `id`? Surprising difference to the other objects.